image_width: 640
image_height: 480
camera_name: usb_cam
camera_matrix:
  rows: 3
  cols: 3
  data: [445.10792629,   0.0 ,337.72928031,  0.0 ,  445.48201089 ,246.31395522,  0.0,0.0,1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [ 0.11847053, -0.35759139 , 0.00148395,  0.00276497 , 0.61140408]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0.0, 0.0, 0.0, 1, 0.0, 0.0, 0.0, 1]
projection_matrix:
  rows: 3
  cols: 4
  data: [445.10792629,   0.0 ,        337.72928031, 0.0, 445.48201089 ,246.31395522, 0.0, 0.0, 1.0, 0.000000, 1.000000, 0.000000]